### ISSUE TITLE
Kingdom Hearts cutscene drifting off

### ISSUE THREAD

**Issue by edwardmx96** [jpd002/Play-#496]
Hey, I have been testing Kingdom Hearts, it runs really well, maybe 26 fps average in-game on OnePlus 3. I can't get past the first boss battle which is against Sora's shadow. After I beat Sora's shadow, the cutscene plays for a bit and for some reason the cutscene starts to drift off on the screen and starts swirling. It's hard to explain, but basically the game gets stuck at the swirling part and I can't get past it.

**Comment by edwardmx96** [comment]
And how do I upload screenshots on here? It would make things easier. One more thing, I tested Kingdom Hearts 2 and it crashes after the title screen loads. It used to work before and I could go in-game with about 10-12 fps, but not sure what it going on. Thanks in advance.

**Comment by edwardmx96** [comment]
![screenshot_20161118-041847](https://cloud.githubusercontent.com/assets/12591131/20554068/3b88a9f4-b10e-11e6-8b6a-3318d8663713.png)
![screenshot_20161118-042135]
(https://cloud.githubusercontent.com/assets/12591131/20554082/4dcb8a0a-b10e-11e6-9ac3-4a77da5a9916.png)

Runs really well!


![screenshot_20161122-233120](https://cloud.githubusercontent.com/assets/12591131/20554095/6b66cb74-b10e-11e6-9a6d-c66efb1286aa.png)
![screenshot_20161122-233152](https://cloud.githubusercontent.com/assets/12591131/20554099/7c68e36c-b10e-11e6-9305-05155d59e42a.png)

This is where it starts drifting off.

![screenshot_20161122-233102](https://cloud.githubusercontent.com/assets/12591131/20554103/83be5caa-b10e-11e6-9203-a76747523559.png)

This is where the cutscene starts swirling endlessly.

![screenshot_20161118-043046]
(https://cloud.githubusercontent.com/assets/12591131/20554109/8b2a41d4-b10e-11e6-9509-2149a756b83f.png)

**Comment by Kygies27** [comment]
Obviously this is all part of the beta. On my Note 4 I can't get past 'Create New Game'.

Have you tried it again with the new update (Dec 2)?

**Comment by edwardmx96** [comment]
Yes, I have tried the new update. Kingdom Hearts 2 loads on my shield tablet now, can go in game. For Kingdom Hearts 1, the emulation speed has gone up maybe 4-7 fps, but it still does the swirling thing after the first boss battle. Both go in game. KH2 gets about 8-20 fps, KH1 gets about 14-39 fps.

**Comment by literalmente-game** [comment]
This issue is outdated, if you can test it again, add all the new information to the new issues:
Kingdom Hearts 1: https://github.com/jpd002/Play-Compatibility/issues/254
Kingdom Hearts 2: https://github.com/jpd002/Play-Compatibility/issues/256
This issue can be closed now
